engine/gamesys/src/gamesys/components/comp_label.cpp

```diff
@@ -543,7 +543,21 @@ namespace dmGameSystem
 
     void CompLabelOnReload(const dmGameObject::ComponentOnReloadParams& params)
     {
-        (void)params;
+        LabelResource* resource = (LabelResource*)params.m_Resource;
+        dmGameSystemDDF::LabelDesc* ddf = resource->m_DDF;
+
+        LabelWorld* label_world = (LabelWorld*)params.m_World;
+        LabelComponent* component = &label_world->m_Components.Get(*params.m_UserData);
+        component->m_Size     = Vector3(ddf->m_Size[0], ddf->m_Size[1], ddf->m_Size[2]);
```

Comment:
Is there perhaps some value in sharing this code with the code in CompLabelCreate() which also sets the component values from a LabelDesc?

Reply:
I think, yes. I'll make additional function with shared code and will reuse it.